pkg/ratelimiter/ratelimiterset.go

```diff
@@ -0,0 +1,46 @@
+package ratelimiter
+
+import (
+	"github.com/bookingcom/nanotube/pkg/conf"
+	"github.com/bookingcom/nanotube/pkg/metrics"
+	"sync"
+	"time"
+)
+
+// Set consists of all the RateLimiters in use.
+type Set struct {
```

Comment:
This data type is unnecessary if we remove the lock and the map. See other comments for details.